mayastor/src/core/mod.rs

```diff
@@ -78,3 +78,9 @@ pub enum CoreError {
         len: usize,
     },
 }
+
+// Which kind of target interface to use for a bdev
```

Comment:
This does not belong here. We should move it to ../target

Reply:
Will move to target.